### PR TITLE
Remove all but 5 last backups of .arkcase folder

### PR DESCRIPTION
**Pull request by dzdimov**

### CODE REVIEW

File: vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml [hunk 1]
```diff
@@ -20,6 +20,11 @@
   set_fact:
     config_backup_folder_path: "{{ root_folder }}/data/arkcase-home/.arkcase-backup-{{ ansible_date_time.iso8601 }}"
 
+- name: Delete all but last 5 backups
+  become: yes
+  become_user: arkcase
+  shell: ls -atp | grep ".arkcase-backup*"| tail -n +6 | xargs -d '\n' -r rm -rf
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml]: @dzdimov , what will happen if the tail output is empty, like when there are fewer than 5 previous backups?

Comment (david-oc-miller) [reply vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml]: Also, the following command (which creates a new backup of the folder) specifies the starting folder "{{ root_folder }}/data/arkcase-home/", I think for consistency we should have the same starting folder in this command.

Comment (bojmil-armedia) [reply vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml]: No issue when there are fewer than 5 previous backups. We tested it.
We fixed the issue with starting folder. Thanks for that one.

File: vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml [hunk 2]
```diff
@@ -20,6 +20,14 @@
...
     config_backup_folder_path: "{{ root_folder }}/data/arkcase-home/.arkcase-backup-{{ ansible_date_time.iso8601 }}"
 
+- name: Delete all but last 5 backups
+  become: yes
+  become_user: arkcase
+  shell: |
+         cd {{ root_folder }}/data/arkcase-home
+         ls -atp | grep ".arkcase-backup*"| tail -n +6 | xargs -d '\n' -r rm -rf
+  when: arkcase_config_path.stat.exists
+
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml:30]: Two other approaches will be more ansible-like and easier to read:

```yaml
shell: ls -atp ....
args:
  chdir: "{{ root_folder}}..."
```

or: 

```yaml
shell: ls -atp {{ root_folder }}/data/arkcase-home | grep ".arkcase-backup*" ...
```

Comment (david-oc-miller) [inline vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml:30]: Two other approaches will be more ansible-like and easier to read:

```yaml
shell: ls -atp ....
args:
  chdir: "{{ root_folder}}..."
```

or: 

```yaml
shell: ls -atp {{ root_folder }}/data/arkcase-home | grep ".arkcase-backup*" ...
```

Comment (bojmil-armedia) [reply vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml:30]: Done. Thanks.

Comment (bojmil-armedia) [reply vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml:30]: Done. Thanks.

**Pull request merged**